src/stringfilter.cpp

```diff
@@ -116,6 +116,19 @@ void StringFilter::AddLine(const char *str)
 	}
 }
 
+/**
+ * Pass another text line from the current item to the filter.
+ *
+ * You can call this multiple times for a single item, if the filter shall apply to multiple things.
+ * Before processing the next item you have to call ResetState().
+ *
+ * @param string Another line from the item.
```

Comment:
Does not match the parameter; I'd suggest going for `str` as then there is no clash with the class name.

Maybe we should get a GitHub work flow that checks the doxygen logs for missing/extra comments, so it can warn about these inconsistencies automatically.

Reply:
Yep, no excuse on this or the missing doxygen elsewhere besides sloppy self-review. Sorry about that. 🙂